packages/harpguru-core/src/utils/reduce-celltogglebuffer-to-harpstrata/reduce-celltogglebuffer-to-harpstrata.ts

```diff
@@ -4,18 +4,18 @@ import type { DegreeIds } from 'harpparts'
 
```

Comment:
Arguably this could have a check for harpstrata equality before return. See other code review comments.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/120#issuecomment-943034083